src/org/usfirst/frc/team694/robot/commands/CrabArmFlapOutCommand.java

```diff
@@ -0,0 +1,23 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import org.usfirst.frc.team694.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class CrabArmFlapOutCommand extends Command {
+
+    public CrabArmFlapOutCommand() {
+        requires(Robot.crabArm);
+    }
+
+    protected void execute() {
+        Robot.crabArm.flapOut();
+    }
```

Comment:
This is definitely not going to work. There are quite a few things, but the biggest one is that this will run the motors in the opposite direction at full speed forever and stall out the motors or break the crab-arms.